bitnami/hubble/templates/relay/cert.yaml

```diff
@@ -0,0 +1,40 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.tls.enabled .Values.tls.autoGenerated.enabled (eq .Values.tls.autoGenerated.engine "cert-manager") }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ printf "%s-crt" (include "hubble.relay.fullname" .) }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    app.kubernetes.io/part-of: hubble
+    app.kubernetes.io/component: relay
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  secretName: {{ printf "%s-crt" (include "hubble.relay.fullname" .) }}
```

Comment:
Shouldn't it be?

```suggestion
  secretName: {{ include "hubble.tls.relay.secretName" . }}
```

Reply:
Optional but not mandatory, that template is returning the same value if an existing secret isn't provided.

Reply:
done at https://github.com/bitnami/charts/pull/26579/commits/e6b5616dc0f607c5df45186f504b1588c96ed484